pyproject.toml

```diff
@@ -3,16 +3,41 @@ requires = ["hatchling", "hatch-requirements-txt"]
 build-backend = "hatchling.build"
 
 [project]
-name = "OTAnalytics"
-version = "0.0.1"
-dynamic = ["dependencies"]
+name = "OTAnalytics-EXAMPLE-TEST-DISTRIBUTION"
+dynamic = ["dependencies", "version"]
+authors = [
```

Comment:
Should we use a generic `{ name="OpenTrafficCam Contributors", email="team@opentrafficcam.org" }`?